README.rst

```diff
@@ -113,9 +113,17 @@ Versioning and releases
 
 This library has its version automatically updated by Lerna (i.e., ``lerna version``) using semantic-versioning under-the-hood when the release is published to npm. Lerna is configured to use independent versioning with conventional commits, as opposed to keeping all package versions in sync.
 
-When a PR is merged, Lerna creates a release commit (e.g., ``chore(release): publish``). In this commit, Lerna increments the versions in the appropriate package.json files for any changed packages, creates Git tags, and updates the CHANGELOG file.
+When a PR is merged, you must manually run Lerna to create a release commit (e.g., ``chore(release): publish``). In this commit, Lerna increments the versions in the appropriate package.json files for any changed packages, creates Git tags, and updates the CHANGELOG file.
+To create this commit:
+::
+
+    npx lerna@6 version --conventional-commits --create-release github --no-push
```

Comment:
nit/suggestion: we could create a new NPM script in `package.json` such that running `npm run create-release` or similar is aliased to `npx lerna@6 version --conventional-commits --create-release github --no-push` so these exact arguments aren't needed each release.

The docs could just say say `npm run create-release` + the `git push` commands after.